.nvmrc

```diff
@@ -1 +1 @@
-6.7.0
+lts/boron
```

Comment:
Perhaps it expects a number version? Travis version of nvm could be different from yours.. I have not really used nvm before either, so I am not sure.